LFS171x/fabric-material/basic-network/README.md

```diff
@@ -11,6 +11,8 @@ To stop it, run ``stop.sh``
 To completely remove all incriminating evidence of the network
 on your system, run ``teardown.sh``.
 
+This source code is based on (https://github.com/hyperledger/fabric-samples). Big thanks to the Hyperledger Team and their [Maintainers](http://hyperledger-fabric.readthedocs.io/en/latest/MAINTAINERS.html).
```

Comment:
Hyperledger --> Hyperledger Fabric

Reply:
Is this based on or pretty much a direct copy? I remember that this seemed like it was a direct copy when I did the diff.

Reply:
rephrased and edited to say : "This code is from..."